src/components/Patient/PatientDetailsTab/ShiftingHistory.tsx

```
@@ -0,0 +1,260 @@
+                      </div>
+                      <div className="sm:col-span-1">
+                        <dt
+                          title=" Origin facility"
+                          className="flex items-center text-sm font-semibold leading-5 text-zinc-400"
+                        >
+                          <CareIcon
+                            icon="l-plane-fly"
+                            className="mr-2 text-lg"
+                          />
+                          <dd className="text-sm font-bold leading-5 text-secondary-900">
+                            {(shift.origin_facility_object || {})?.name}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)